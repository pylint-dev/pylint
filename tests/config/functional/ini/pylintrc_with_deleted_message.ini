# Check that we raise an informational when a deleted messages exists in a .pylintrc file
# See https://github.com/PyCQA/pylint/issues/6794
[messages control]
disable = logging-not-lazy, buffer-builtin
enable = useless-option-value, locally-disabled, cmp-builtin
jobs = 10
reports = yes
