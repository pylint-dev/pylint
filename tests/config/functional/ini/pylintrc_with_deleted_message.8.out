************* Module {abspath}
{relpath}:1:0: R0022: Useless option value for '--disable', 'buffer-builtin' was removed from pylint, see https://github.com/PyCQA/pylint/pull/4942. (useless-option-value)
{relpath}:1:0: R0022: Useless option value for '--enable', 'cmp-builtin' was removed from pylint, see https://github.com/PyCQA/pylint/pull/4942. (useless-option-value)
